fields/types/sortableRelateds/SortableRelateds.js

```diff
@@ -113,30 +109,28 @@ module.exports = Field.create({
     });
   },
 
-  loadOptions(postIds) {
+  loadOptions(input, callback) {
     const filters = this.buildFilters();
     xhr({
-      url: Keystone.adminPath + '/api/' + this.props.refList.path + '?basic&' + filters,
+      url: Keystone.adminPath + '/api/' + this.props.refList.path + '?basic&search=' + input + '&' + filters,
       responseType: 'json',
     }, (err, resp, data) => {
       if (err) {
         console.error('Error loading items:', err);
-        return;
+        return callback(null, []);
```

Comment:
Question: Why does `callback` here return `null` instead of `err` ?

Reply:
thanks for spotting this, fixed.